loading.bs

```diff
@@ -331,6 +385,376 @@ Note: Applying the signed exchange's response here has the effect of letting a
+            1. For each |sxg| of |navigationParams|'s [=navigation
+                params/prefetched subresource signed exchanges=]:
+                1. If |headerIntegrity| is the same as the value of |sxg|'s
+                    [=exchange/response=]'s [=response/header integrity value=]
+                    encoded as a [[CSP]] <a grammar>hash-source</a>, then set
+                    |prefetched alternate exchange| to |sxg|.
+            1. If |headerIntegrity| is not null and |prefetched alternate
+                exchange| is null, then set |canLoadAlternateSxg| to false.
+                
+                Note: This means that there is a matching
+                <{link/rel/allowed-alt-sxg}> link for the <{link/rel/preload}>
+                |link|, but the matching signed exchange has not been
+                prefetched. In this case the UA MUST NOT use the prefetched
+                subresource signed exchanges for other  <{link/rel/preload}>
+                links. This is intended to prevent the referrer page from
+                encoding a tracking ID into the set of subresources it
+                prefetches.
+            1. [=list/Append=] the [=alternate signed exchange peload info=]
+                (|link|, |linkTarget|, |prefetched alternate exchange|) to
+                |preloadLinkItems|.
+        1. For each |preloadLinkItem| of |preloadLinkItems|:
+            1. Let |corsAttribute| be the [=CORS settings attribute=] of
+                |preloadLinkItem|'s [=alternate signed exchange peload
+                info/link=]'s [=Target Attribute named=] `"crossorigin"`.
+            1. Let |request| be the result of [=creating a potential-CORS
+                request=] given |preloadLinkItem|'s [=alternate signed exchange
+                peload info/target=], the empty string, and |corsAttribute|.
+            1. Set |request|'s [=request/synchronous flag=].
+            1. Set |request|'s [=request/client=] to |document|.
+            1. Set |request|'s [=request/cryptographic nonce metadata=] to
+                |preloadLinkItem|'s [=alternate signed exchange peload
+                info/link=]'s [=Target Attribute named=] `"nonce"`.
+            1. Set |request|'s [=request/integrity metadata=] to
+                |preloadLinkItem|'s [=alternate signed exchange peload
+                info/link=]'s [=Target Attribute named=] `"integrity"`.
+            1. Set |request|'s [=request/referrer policy=] to be the [=referrer
+                policy attribute=] of |preloadLinkItem|'s [=alternate signed
+                exchange peload info/link=]'s [=Target Attribute named=]
+                `"referrerpolicy"`.
+            1. Set |request|'s [=request/destination=] to |preloadLinkItem|'s
+                [=alternate signed exchange peload info/link=]'s [=Target
+                Attribute named=] `"as"`.
+            1. If |canLoadAlternateSxg| is true, then set |request|'s
```

Comment:
I'm not sure I get this. It seems that the resource is loaded even if `canLoadAlternateSxg` is false.
If that's intentional, a better name could clarify that

Reply:
Added Note.